firefox_ui_tests/remote/security/test_security_notification.py

```diff
@@ -0,0 +1,69 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from marionette import By
+from marionette.errors import MarionetteException
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestSecurityNotification(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.test_data = [
+            # Invalid cert page
+            'https://summitbook.mozilla.org',
+            # Secure page
+            'https://ssl-ev.mozqa.com/',
+            # Unsecure page
+            'http://www.mozqa.com'
+        ]
+
+        self.identity_box = self.browser.navbar.locationbar.identity_popup.box
```

Comment:
Sorry, my failure. This is fine, given that it's a Marionette HTMLElement instance.